### ISSUE TITLE
Grunt doesn't watch SCSS files

### ISSUE THREAD

**Issue by cameronjonesweb** [MatterSolutions/matter-kit#7]
Not sure why... or if it's just certain directories but definitely not watching components

**Comment by KawaiiHannah** [comment]
So I think it might have something to do with being shared files with Chassis/vagrant, but I'm not entirely certain on that. Gonna have to investigate it further.

**Comment by cameronjonesweb** [comment]
Seems to be inconsistent as to when it watches files like it's got ADHD or something

**Comment by KawaiiHannah** [comment]
Yeah, I can't get it to watch any Sass files at all, whether they're in components or assets/css/scss
